cmd/dep/testdata/harness_tests/ensure/update/novendor/initial/Gopkg.lock

```diff
@@ -0,0 +1,15 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  name = "github.com/sdboyer/deptest"
+  packages = ["."]
+  revision = "dummy"
+  version = "dummy"
```

Comment:
Let's set this to some proper values from https://github.com/sdboyer/deptest/releases.